### PR TITLE
fix private features' link in classes overview page

### PR DESCRIPTION
**Pull request by Ahmed-Hakeem**
link hash should be replaced either by "#private_properties_2" or like what I did here.

I believe linking to the private fields page (like what I've done here) is better than linking to the private properties section in the same page.

In addition, I think the header having the "2" in its id because there's already another html element without "2" in the  same document.

### CODE REVIEW

File: files/en-us/web/javascript/reference/classes/index.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ class Rectangle {
 }
 ```
 
-Class fields are similar to object properties, not variables, so we don't use keywords such as `const` to declare them. In JavaScript, [private features](#private_class_features) use a special identifier syntax, so modifier keywords like `public` and `private` should not be used either.
+Class fields are similar to object properties, not variables, so we don't use keywords such as `const` to declare them. In JavaScript, [private features](/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties#syntax) use a special identifier syntax, so modifier keywords like `public` and `private` should not be used either.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/classes/index.md:182]:
```suggestion
Class fields are similar to object properties, not variables, so we don't use keywords such as `const` to declare them. In JavaScript, [private features](#private_class_features_2) use a special identifier syntax, so modifier keywords like `public` and `private` should not be used either.
```

**Review by Josh-Cena — APPROVED** [review]
Good spot, but I'm still going to use the `_2` anchor, because this section always uses an internal anchor where available, and internal anchors are usually preferable to page links because they are the easiest to navigate around.

**Comment by Ahmed-Hakeem** [comment]
Oh, it should be like what I've mentioned in the description "#private_properties_2" not "#private_class_features_2" ..... the name of it is different as well @Josh-Cena.

**Comment by Josh-Cena** [comment]
Ooops...

**Pull request merged**